.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "python.pythonPath": "/usr/bin/python2.7"
```

Comment:
I could not manage to use python 3, e.g. I could not install pyaudio for this version…